src/store/authentication/index.ts

```diff
@@ -0,0 +1,28 @@
+import { Payload } from './saga';
+import { createAction, createSlice, PayloadAction } from '@reduxjs/toolkit';
+
+export enum SagaActionTypes {
+  Authorize = 'authentication/saga/authorize',
+}
+const authorize = createAction<Payload>(SagaActionTypes.Authorize);
+
+interface AuthenticationState {
+  accessToken: string;
+}
+const initialState: AuthenticationState = {
+  accessToken: null,
+};
+
+const slice = createSlice({
+  name: 'authentication',
+  initialState,
+  reducers: {
+    setAccessToken: (state, action: PayloadAction<AuthenticationState['accessToken']>) => {
+      state.accessToken = action.payload;
```

Comment:
Should we store the access token in a cookie or local storage? If it's in state will the user be prompted to login when opening a new tab?

Reply:
You are right, as far as I know the cookie should come in the response from the api

another comment which is related to the token cookie here: https://github.com/zer0-os/zOS/pull/164#discussion_r969335388

Reply:
(yeah. if we can use a cookie and avoid storing this, even better.)